README.md

```diff
@@ -502,6 +503,50 @@ echo "Hello " . $snapshot['firstName'];
 $ composer require google/cloud-firestore
 ```
 
+## Google Cloud Asset (Beta)
+
+- [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/latest/asset/assetclient)
+- [Official Documentation](https://cloud.google.com/resource-manager/docs/cai/)
```

Comment:
this is a 404 -- is the page just not published yet?

Reply:
Yeah it's still behind the whitelist filter